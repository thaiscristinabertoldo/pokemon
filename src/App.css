body {
  margin: 0px;
  background-color: #fcfcfc;
}

form {
  width: 100%;
}

::-webkit-scrollbar {
  width: 8px;
}

::-webkit-scrollbar-track {
  background: #f1f1f1;
}

::-webkit-scrollbar-thumb {
  background: #ffcc01;
}
