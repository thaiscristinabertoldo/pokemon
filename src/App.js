import React, { Suspense } from 'react'

import { BrowserRouter } from 'react-router-dom'

import { ApolloProvider } from '@apollo/react-hooks'

import { Loading } from './components/Loading'
import { PageLayout } from './components/PageLayout/PageLayout'

import apolloClient from './api/apollo'
import Pokemon from './pages/Pokemon'

import './App.css'

const App = () => {
  return (
    <ApolloProvider client={apolloClient}>
      <BrowserRouter>
        <PageLayout>
          <Suspense fallback={<Loading />}>
            <Pokemon />
          </Suspense>
        </PageLayout>
      </BrowserRouter>
    </ApolloProvider>
  )
}

export default App
