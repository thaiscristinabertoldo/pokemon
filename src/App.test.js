import React from 'react'
import { render, cleanup } from '@testing-library/react'
import App from './App'

afterEach(cleanup)

describe('Take a snapshot', () => {
  it('should take a snapshot', () => {
    const { asFragment } = render(<App />)

    expect(asFragment(<App />)).toMatchSnapshot()
  })
})
