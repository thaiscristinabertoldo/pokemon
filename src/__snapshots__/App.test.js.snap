// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Take a snapshot should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-AxirZ jkHhiD"
    data-testid="navbar"
  >
    <a
      data-testid="home-link"
      href="/pokemon"
    >
      <img
        alt="PokÃ©mon"
        class="img"
        height="50px"
        src="logo.png"
      />
    </a>
  </div>
  <div
    class="sc-AxiKw gsSvlJ"
  >
    <div
      class="sc-AxjAm ekzoek"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-spinner fa-w-16 fa-spin fa-5x "
        data-icon="spinner"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M304 48c0 26.51-21.49 48-48 48s-48-21.49-48-48 21.49-48 48-48 48 21.49 48 48zm-48 368c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm208-208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zM96 256c0-26.51-21.49-48-48-48S0 229.49 0 256s21.49 48 48 48 48-21.49 48-48zm12.922 99.078c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48c0-26.509-21.491-48-48-48zm294.156 0c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48c0-26.509-21.49-48-48-48zM108.922 60.922c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.491-48-48-48z"
          fill="currentColor"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
